goodDeeds/app/src/test/java/com/goodpeople/gooddeeds/Controller/AccountControllerTest.java

```diff
@@ -73,5 +73,8 @@ public void shouldUpdateAccountDetails() {
         Assert.assertEquals(accountController.accountHandler().getPostalCode(), 12345);
     }
 
-
+    @Test
+    public void logout() {
+        accountController.logout();
+    }
```

Comment:
have to test something? maybe assertEquals(false, accountController.isLoggedIn()) or equal?

Reply:
Hahaha, yes that's true